library/common/http/dispatcher.cc

```diff
@@ -0,0 +1,439 @@
+#include "library/common/http/dispatcher.h"
```

Comment:
This file should be gone, right?

Reply:
Yes it looks like something got screwed up in the merge.